src/overlays/actors/ovl_Bg_Spot15_Rrbox/z_bg_spot15_rrbox.c

```diff
@@ -329,9 +330,8 @@ void func_808B43D0(BgSpot15Rrbox* this, PlayState* play) {
     Actor_MoveXZGravity(actor);
 
     if (actor->world.pos.y <= BGCHECK_Y_MIN + 10.0f) {
-        // "Lon Lon wooden crate fell too much"
-        PRINTF("Warning : ロンロン木箱落ちすぎた(%s %d)(arg_data 0x%04x)\n", "../z_bg_spot15_rrbox.c", 599,
-               actor->params);
+        PRINTF("Warning : " T("ロンロン木箱落ちすぎた", "Lonlon Wooden Box fell too far") "(%s %d)(arg_data 0x%04x)\n",
+               "../z_bg_spot15_rrbox.c", 599, actor->params);
```

Comment:
Tiny, but I think its fine to make it proper "Lon Lon" as in the name of the place. The japanese language doesnt have capitalization or spaces in the same way, its not a deliberate difference that needs to be preserved